frontend/src/deployment/metadata.ts

```diff
@@ -1,63 +1,96 @@
 import fs from 'fs';
 import path from 'path';
-import { LanguageSupport } from '../types/RuleMetadata';
 
+import { LanguageSupport } from '../types/RuleMetadata';
 import { getRulesDirectories, listSupportedLanguages } from './utils';
 
+/**
+ * Save the given metadata to disk.
+ */
+function writeRuleMetadata(dstDir: string, filename: string, metadata: any) {
+  const file = path.join(dstDir, filename);
+  fs.writeFileSync(file, JSON.stringify(metadata), { encoding: 'utf8' });
```

Comment:
```suggestion
  fs.writeFileSync(file, JSON.stringify(metadata, null, 2), { encoding: 'utf8' });
```
I agree with keeping the `rule-index-store.json` short - with no line breaks or indentation. However, I think we should keep the metadata for each rule readable, in case some want's to inspect the `language-metadata.json` file, or download it somewhere else.
These files are relatively short, so I don't think this adds much overhead, but substantially improves readability.

Reply:
Currently, AFAICT, those JSON files are not easily downloadable. At least I couldn't find links to them (although if you know where they are you can handcraft the URLs). So I wouldn't expect people to download them. If someone really wants them, they can easily reformat them too (using `python -m json.tool` for example).

In numbers, I just measured a reduction of 30%: before we had 10.14MB of `*-metadata.json` files, now about 7.03MB.

The readability aspect isn't modified by this change: they are pretty printed anyway in `RulePage.tsx` anyway.